assets/wire/actions.ts

```diff
@@ -506,6 +507,36 @@ export function reloadMyTopics(reloadTopic: any = false) {
     };
 }
 
+export const UPDATE_FOLDERS = 'UPDATE_FOLDERS';
+
+/**
+ * @param {bool} global - fetch company or user folders
+ * @param {bool} skipDispatch - if true it won't replace folders in store
+ */
```

Comment:
it looks like comments from some other function? where are the params that are being commented?